src/assets/usage-examples-code/sprites/free_sprite/free_sprite-3-free-sprite.cpp

```diff
@@ -0,0 +1,42 @@
+/**
```

Comment:
Remove this comment at the top, its not needed in the program code thats displayed.

Reply:
I have removed the comments